terraform/environments/bootstrap/member-bootstrap/account.tf

```diff
@@ -0,0 +1,19 @@
+resource "aws_account_alternate_contact" "infrastructure_support_operations" {
+
+  alternate_contact_type = "OPERATIONS"
+
+  name          = "Infrastructure Support"
+  title         = "Infrastructure Support Operations"
+  email_address = local.application_tags.infrastructure-support
+  phone_number  = "+0000000000"
+}
+
+resource "aws_account_alternate_contact" "infrastructure_support_security" {
+
+  alternate_contact_type = "SECURITY"
```

Comment:
The `SECURITY` contact is already managed by [aws-root-account](https://github.com/ministryofjustice/aws-root-account/blob/main/management-account/terraform/alternate-contacts.tf#L10) for _all_ AWS accounts, so you won't need this here.

It is periodically run (manually) to capture any new accounts created 👍🏼

Reply:
Thanks! Removed :D